clearnode/custody.go

```diff
@@ -628,7 +636,19 @@ func (c *Custody) UpdateBalanceMetrics(ctx context.Context, assets []Asset, metr
 	logger.Debug("open channels metric updated", "network", c.chainID, "channels", count)
 }
 
+// saveContractEvent saves a contract event to the database if it has not been processed before.
+// It returns true if the event was already processed, false otherwise.
```

Comment:
The `saveContractEvent` function that this comment is referring to does not return a boolean.
Please resolve this confusion.